Makefile

```diff
@@ -0,0 +1,42 @@
+
+.DEFAULT_GOAL := help
+
+
+.PHONY: build-generator, clean, openapi-generator-cli help
+
+JDK_LINK := https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.7%2B7/OpenJDK17U-jdk_x64_linux_hotspot_17.0.7_7.tar.gz
+JDK_DIR := $(CURDIR)/.jdk
+OPENAPI_GENERATOR_CLI := $(CURDIR)/modules/openapi-generator-cli/target/openapi-generator-cli.jar
+
+
+help: ## help on rule's targets
+ifeq ($(IS_WIN),)
+	@awk --posix 'BEGIN {FS = ":.*?## "} /^[[:alpha:][:space:]_-]+:.*?## / {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
```

Comment:
I think you do not need this part. you do not check for windows and anyway we only use linux